.github/scripts/update-release-notes.js

```diff
@@ -73,19 +80,19 @@ var optionsget = {
         'User-Agent': 'server-side',
       }
 };
-
-var reqGet = https.request(optionsget, function(res) {
-    // console.log("statusCode: ", res.statusCode);
- 
+debugLog(`request options: ${JSON.stringify(options)}`)
+var reqGet = https.request(options, function(res) {
+    debugLog(`response statusCode: ${res.statusCode}`)
     res.on('data', function(d) {
         str = new TextDecoder("utf-8").decode(d)
+        debugLog(`response json string: ${str}`)
         responseJson = JSON.parse(str)
 
         responseJson.response.docs.forEach(element => {
-            if (isEarlierThan24Hours(element.timestamp) == false) {
+            if (isEarlierThanXHours(checkReleaseWithinHours, element.timestamp) == false) {
```

Comment:
In the long term, we should maintain a list of versions for which release notes have been published and generate new notes based off of it.